src/app/app.routes.ts

```diff
@@ -1,7 +1,6 @@
 import { Routes } from "@angular/router";
-import { TestComponent } from "./test/test.component";
 
 export const APP_ROUTES: Routes = [
   { path: '', pathMatch: 'full', redirectTo: 'test' },
-  { path: 'test', component: TestComponent },
+  { path: 'test', loadChildren: () => import('./test/test.module').then(m => m.TestModule) },
```

Comment:
The "shell" app doesn't actually care about this. 
Just to demonstrate that you have to expose a "lazyloadable" module